### ISSUE TITLE
Barbican takeover on the Ceilometer and Aodh pages

### ISSUE THREAD

**Issue by madmax-codewarrior-0** [OpenStackweb/openstack-org#293]
This might be a small thing, but it appears we have a perpetrated pilfering by a persistent purple porcupine for the Telemetry projects. 

Looks like the mascot on the Ceilometer software page (https://www.openstack.org/software/releases/yoga/components/ceilometer), Aodh page (though I'm not sure if it ever had a mascot besides the meerkat; https://www.openstack.org/software/releases/yoga/components/aodh), and the Telemetry development/support team's mascot as well, have all been substituted with Barbican's mascot. 

I don't have the knowledge to hunt it down very far, but I at least I got to the page template and the line that is suppose to fill in the mascot name and pull a reference to the mascot:

https://github.com/OpenStackweb/openstack-org/blob/4e79b5056f927c31ab34f16bd7dd004cadbca753/software/templates/Layout/SoftwareHomePage_getComponent.ss#L15

https://github.com/OpenStackweb/openstack-org/blob/4e79b5056f927c31ab34f16bd7dd004cadbca753/software/templates/Layout/SoftwareHomePage_getComponent.ss#L97

I'm guessing these two classes might contain something helpful:

https://github.com/OpenStackweb/openstack-org/blob/master/software/code/infrastructure/active_records/Mascot.php

https://github.com/OpenStackweb/openstack-org/blob/master/software/code/infrastructure/active_records/OpenStackComponent.php

And possibly this database migration class:

https://github.com/OpenStackweb/openstack-org/blob/master/software/code/migrations/PopulateMascotsMigration.php

Another thought at the back of my head is that the "Telemetry" project has both Ceilometer and Aodh, but I believe the subprojects/deliverables don't have their own individual mascot.

**Comment by iamweswilson** [comment]
Good catch, thanks for letting us know! I have copied Josh Lohse, who manages the web properties for the Foundation. He should be able to help us get the mascots sorted and back in the right place. 

Wes

**Comment by iamweswilson** [comment]
That is a fantastic catch, and great detective work!
 
I'll dig into this and get the mascot logos sorted, and also make sure those project pages are purged of the porcupine logo problem. :)
 
Thank you!
 
Josh
 
-----Original Message-----

**Comment by madmax-codewarrior-0** [comment]
You betcha! Happy to help. 

A small piece of my thoughts said it could have been in the DB or something went wrong in the migration. If not something DB related and if I had more time on my hands, I would have dug a little further and possibly put in a PR.
